charts/core/src/ChartCard/ChartCard.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { axe } from 'jest-axe';
+
+import { ChartCard } from './ChartCard';
+import { ChartCardProps } from './ChartCard.types';
+
+const defaultContentText = 'shenanigans';
+
+const defaultProps = {
+  title: 'test',
+  headerContent: <div>{defaultContentText}</div>,
+  children: 'Chartchartchartchartchart',
+};
+
+const renderChartCard = (props?: Partial<ChartCardProps>) =>
+  render(<ChartCard {...defaultProps} {...props} />);
+
+const clickToggleButton = () => {
+  userEvent.click(screen.getByLabelText('Toggle button'));
```

Comment:
nit: Thoughts on using a testid here? Also, will we be providing test utils for this component? If so, we have a pattern of creating a constant file to store testids. For example: https://github.com/mongodb/leafygreen-ui/blob/main/packages/select/src/constants.ts

Reply:
Thanks for clarifying!